lib/ssl/test/ssl_crl_SUITE.erl

```diff
@@ -251,6 +257,17 @@ crl_verify_valid(Config) when is_list(Config) ->
 
     crl_verify_valid(Hostname, ServerNode, ServerOpts, ClientNode, ClientOpts),
 
+    ssl_crl_cache:insert("http://foobar/erlangCA/crl.pem", {file, filename:join([PrivDir, "otpCA", "crl.pem"])}),
+
+    R1 = ssl_crl_cache:lookup(#'DistributionPoint'{distributionPoint =
+                                                       {fullName, [{uniformResourceIdentifier, "http://foobar/erlangCA/crl.pem"}]}},
+                              undefined, {{CRLCache, internal_dummy}, internal_dummy}),
+    R2 = ssl_crl_cache:lookup(#'DistributionPoint'{distributionPoint =
+                                                       {fullName, [{uniformResourceIdentifier, "http://localhost/erlangCA/crl.pem"}]}},
+                              undefined, {{CRLCache, internal_dummy}, internal_dummy}),
+    %% Check that same path in URI does not evaluate to same result
+    true = R1 =/= R2,
+
     %% check that delete WITH URI works as well.
     ssl_crl_cache:delete("http://localhost/erlangCA/crl.pem").
```

Comment:
not sure about other tests content. but maybe all items should be deleted, and cache lookup functionality should be verified after removal.

in this particular line, why there is not pattern match? how do you know delete operation succeeded?

ignore if comment is not relevant.

Reply:
I add a ok match it improves the test, although this has nothing to do with my PR it was already part of the test case that I extended.